src/libponyrt/mem/alloc.c

```diff
@@ -28,7 +28,7 @@ void* ponyint_virt_alloc(size_t bytes)
     MAP_PRIVATE | MAP_ANONYMOUS | MAP_NORESERVE, -1, 0);
 #elif defined(PLATFORM_IS_MACOSX)
   p = mmap(0, bytes, PROT_READ | PROT_WRITE,
-    MAP_PRIVATE | MAP_ANON | VM_FLAGS_SUPERPAGE_SIZE_ANY, -1, 0);
```

Comment:
I believe we still need this.

Reply:
That specific flag is not supported on the M1. I believe we'll need to add some `define` magic for this.

Reply:
Yup. That's my understanding as well.